gps-rtk-serial-no-network/gps_rtk_serial_no_network.go

```diff
@@ -0,0 +1,413 @@
+package gpsrtkserialnonetwork
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"io"
+	"math"
+	"sync"
+
+	"github.com/edaniels/golog"
+	"github.com/go-gnss/rtcm/rtcm3"
+	"github.com/golang/geo/r3"
+	slib "github.com/jacobsa/go-serial/serial"
+	geo "github.com/kellydunn/golang-geo"
+	"go.viam.com/rdk/components/movementsensor"
+	"go.viam.com/rdk/components/movementsensor/gpsnmea"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/spatialmath"
+	"go.viam.com/utils"
+)
+
+var Model = resource.NewModel("viam-labs", "movement-sensor", "gps-rtk-serial-no-network")
+
+const serialStr = "serial"
+
+type Config struct {
+	SerialNMEAPath           string `json:"serial_NMEA_path"`
+	SerialNMEABaudRate       int    `json:"serial__NMEA_baud_rate,omitempty"`
```

Comment:
just fyi I changed these config names to make it clear which one is correction path and which one is nmea path, but if we want it to match the scope i'm fine with changing it back

Reply:
that's fine, all lowercase please.